### DISCUSSION TITLE
No output from password prompts (consider outputting ****)

### DISCUSSION THREAD

**Discussion by Torxed** [archlinux/archinstall#288] [Ideas]
I'm not necessarily for or against this, but I'm throwing it up here for discussion.
Would be be beneficial to output something at least when entering things into a password prompt in archinstall?
Or would that be a security concern because people can deduct the length of your password?

Implementing `*` per character written would be easy but there's other options, maybe a rotating `/` character.

**Comment by jtagcat** [comment]
`*` is well known, IIRC Linux Mint used it for sudo pw prompts as well. Even online things (http input type password) use `*` for this.

Rotating `/` rather indicates loading, than hidden.
